config/routes.rb

```diff
@@ -35,6 +35,7 @@
   get 'delivery/login' => 'deliveries#login'
   get 'delivery/index' => 'deliveries#index'
   get 'delivery/claim' => 'deliveries#claim'
```

Comment:
This is late. But it's just occurred to me. Should there also be post requests? Or Patch requests? Since we'll be doing some database updates through the same API?